lib/tools/adb-emu-commands.js

```diff
@@ -2,7 +2,25 @@ import log from '../logger.js';
 import _ from 'lodash';
 
 const PHONE_NUMBER_PATTERN = /^[\+]?[(]?[0-9]*[)]?[-\s\.]?[0-9]*[-\s\.]?[0-9]{2,}$/im;
+
 let emuMethods = {};
+emuMethods.GSM_CALL_ACTIONS = {
+  'GSM_CALL' : 'call',
```

Comment:
As with the other, no need to quote keys in object literal syntax, unless there is some special character (e.g., `'GSM-CALL'` would need them, `GSM_CALL` does not).